File: backend/BakeryStoreApi/DTOs/OrderResponse.cs
namespace BakeryStoreApi.DTOs
{
    public class OrderResponse
    {
        public int OrderId { get; set; }

        public int UserId { get; set; }

        public decimal TotalAmount { get; set; }

        public string? Status { get; set; }

        public DateTime? DeliveryDate { get; set; }

        public DateTime CreatedAt { get; set; }

        public string? PaymentMethod { get; set; }

        public string? PaymentStatus { get; set; }

        public string? ShippingAddress { get; set; }
    }
}

File: app/src/main/java/com/example/bakerystore/models/OrderResponse.kt
package com.example.bakerystore.models

data class OrderResponse(
    val orderId: Int,
    val userId: Int,
    val totalAmount: Double,
    val status: String?,
    val deliveryDate: String?,
    val createdAt: String?,
    val paymentMethod: String?,
    val paymentStatus: String?,
    val shippingAddress: String?
)